examples/anexia-machinedeployment.yaml

```diff
@@ -48,6 +48,30 @@ spec:
               - size: 60
                 performanceType: ENT6
 
+            # each entry in this array will be created as a network interface
+            # of each Machine, connected to the given VLAN
```

Comment:
Nitpicky, but sentences end with a dot and start with a big letter. :p

Reply:
yea yea xD